bin/node-template-spartan/start-farmer.sh

```
@@ -0,0 +1,81 @@
+#!/bin/bash
+setup() {
```

Please add `set -e` to make sure script aborts at the very first error instead of continuing despite errors